model/src/channel/stage_instance.rs

```diff
@@ -1,7 +1,7 @@
 use crate::id::{ChannelId, GuildId, StageId};
 use serde::{Deserialize, Serialize};
 
-#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
```

Comment:
This shouldn't need to implement `Hash`?

Reply:
needs hash for the cache dashmaps

Reply:
Only the `StageId` is used as a key in the map. DashMap (and HashMap, BTreeMap, etc.) only require the key to implement `Hash`: <https://docs.rs/dashmap/4.0.2/dashmap/struct.DashMap.html#impl>

Reply:
actually this is wrong, it needs hash cause the gateway payloads have it. but still that is another question